src/main/java/homework_1/CharCount.java

```diff
@@ -0,0 +1,25 @@
+package homework_1;
+
+public class CharCount {
+    public static final String RED_BOLD = "\033[1;31m";
+
+    public static void main(String[] args) {
+        if (args.length == 0)
+        {
+            System.out.println("There is no parameters, exit program.");
+        }
+        else {
+
+        System.out.println("Count of Char's in words:");
+        for (String arg : args) {
+            if (arg.equals("error")) {
+                System.out.println(RED_BOLD + "Alarm!");
```

Comment:
You need to have close escape string, otherwise all other text will be red color too